docs/water-connection-api-1.0.0.yaml

```
@@ -0,0 +1,364 @@
+    ChannelType:
+      type: object
+      description: Structure for the channel type master
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: System generated identifier for the master data. Will be generated by the MDMS service.
+          readOnly: true
+        channelCode:
+          type: string
+          description: These are user defined constants that will be used by the frontend and backend services to reference this entry. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)